src/sketches/perlin-noise/perlinGraph.js

```diff
@@ -0,0 +1,39 @@
+// Graphing 1D Perlin Noise (1D Noise Graph)
```

Comment:
I've pasted this here, but haven't used it yet or converted it to be more similar to the working radar-sketch.